spec_mode/nirspec_fs.ipynb

```diff
@@ -122,7 +122,9 @@
    "cell_type": "code",
```

Comment:
<p>This was the section I couldn't run without a VPN connection. I think it might be due to missing reference files? </p><p> </p><p>If that's the case, <code>spec_mode_stage_2.ipynb</code> and <code>spec_mode_stage_3.ipynb</code> in this same folder both have instructions in their "Reference Files" sections on how to set those up and make it clear that you can't fully run the notebook outside of the JWebbinar/Institute environment without doing so. I think this notebook could benefit from something similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>